[IMP] Export: Updated quotations template for order lines (#15458)
odoo/documentation: content/applications/essentials/export_import_data.rst

#. When exporting, there is the option to export in two formats: `.csv` and `.xls`. With `.csv`,
   items are separated by a comma, while `.xls` holds information about all the worksheets in a
   file, including both content and formatting.
#. These are the items that can be exported. Use the :icon:`fa-chevron-right` :guilabel:`(Show
   sub-fields)` icon to display more sub-field options. Use the :guilabel:`Search` bar to find
   specific fields. To use the :guilabel:`Search` option more efficiently, click on all the
   :icon:`fa-chevron-right` :guilabel:`(Show sub-fields)` icon to display all fields.

> #. The :icon:`fa-plus` :guilabel:`(Select field)` icon button is present to add fields to the

Reviewer: **Re: UI fallback labels for icons**

It's good you fixed the **List** icon. I'm seeing some others here that I want to bring to your attention to either consider changing or bringing up to your content leader for guidance.

**Background**
The logic for doing the `:icon:`lib-iconName` (fallback label)` structure, when we came up with it originally, was to provide a redundant method that served as an accessibility tool for users in regions with low/no internet access or who were using old tech-- scenarios where icon libraries might not load correctly (or render at all) in the HTML. 

In order for the fallback labels to serve their purpose to enable those users, it means they should follow a standard and consistent formatting, as much as possible. For example a ✏️ icon does not use a fallback label of `(edit)`. It uses `(pencil)`, instead- the name of the visual object- because no matter the region or language, `pencil` = `pencil` and translates nicely (whereas there are multiple words for 'edit' in French, Spanish, Portuguese, etc.)

Generally, whatever the `oi` or `fa` code is in the dev tools console is what we use for the fallback label. There are some exceptions, of course, like the **Actions** menu icon (which has a code of `fa-cog`), because the context provides more clarity than the FA code.

Not asking for changes (can always change later if you want), but I would take a look [here at this appendix](https://docs.google.com/document/d/17UxnITmNyV017LbvqEGmhy6vxICDwkiDuzaE_b40BbE/edit?tab=t.0#heading=h.uo9c9ek7i3ze) for the knowledge-share.

Some examples attached of what I mean, for visual context.

1. is it a "select field" or "plus" icon?
<img width="940" height="616" alt="Image" src="https://github.com/user-attachments/assets/a491d470-5ece-4497-9c26-6faf843af5c3" />

2. is it a "show sub-fields" or "right chevron" icon?
<img width="1362" height="880" alt="Image" src="https://github.com/user-attachments/assets/304f3664-f933-48aa-837c-0d695ac967e6" />

3. is it a "remove field" or is it's a "trash" icon on a field row.
<img width="694" height="78" alt="Image" src="https://github.com/user-attachments/assets/d83d41d5-e675-4bc2-95c3-34f06d797d58" />

   :guilabel:`Fields to export` list.
#. The :icon:`fa-sort` :guilabel:`(Sort)` icon to the left of the selected fields can be used to
   move the fields up and down, to change the order in which they are displayed in the exported
   file. Drag-and-drop using the :icon:`fa-sort` :guilabel:`(Sort)` icon.
#. The :icon:`fa-trash` :guilabel:`(Remove field)` icon is used to remove fields. Click the

>    :icon:`fa-trash` :guilabel:`(Remove field)` icon to remove the field.

Author: Yes, those are the labels that appear when you hover over the Sort and Trash icons.

   Then, use this field's label in the import file to ensure future imports are successful.

.. tip::
   Another useful way to find out the proper column names to import is to export a sample file using

>    the fields that should be imported. This way, if there is not a sample import template, the names

Reviewer 2: ```suggestion
   the fields to be imported. This way, if there is not a sample import template, the names
```
Just might be my personal prefrence, but I like to avoid "should" "will" etc.  (I know "will" is flagged by the Vale, not sure about this one).

   are accurate.

Import from another application
-------------------------------

The :guilabel:`External ID` (ID) is a unique identifier for the line item. Feel free to use one from

> previous software to facilitate the transition to Odoo.

Reviewer 2: ```suggestion
The :guilabel:`External ID` (ID) is a unique identifier for a line item. It is possible to use an :guilabel:`External ID` (ID) from
previous software to facilitate the transition to Odoo.
```
"Feel free" just sounds too "American" to me =D  and something that may not translate well - seems like something difficult ot translate in the right way (that's why I try and use the plainest language for the least possible translation errors. Kind of like why we would never use an idiom or expression that is uniquely English).
